makeStage2.sh

```diff
@@ -39,5 +40,66 @@ cd $usepop/pop/x/src/
 # ways to construct the Poplog system.
 # $usepop/pop/src/newpop -link -nox -norsv
 # $usepop/pop/src/newpop -link -x=-xm -norsv
+# $usepop/pop/src/newpop -link -x=-xt -norsv
+
+mkdir -p ${BUILD_HOME}/environments
```

Comment:
Can we get quotes on all instances of strings that use path variables?

Reply:
Drat. Yes.